Jenkinsfile.oscontainer

```diff
@@ -0,0 +1,23 @@
+def DOCKER_IMG = "registry.fedoraproject.org/fedora:28"
+// Turns out the Jenkins docker stuff totally breaks with SELinux
+def DOCKER_ARGS = "--privileged"
+
+node(env.NODE) {
+    checkout scm
+
+    stage("Prepare Dockerfile") {
+        docker.image(DOCKER_IMG).pull()
+        docker.image(DOCKER_IMG).inside(DOCKER_ARGS) {
+          sh """sed -e 's,@OSTREE_REPO_URL@,${env.OSTREE_INSTALL_URL},' < Dockerfile.rollup.in > Dockerfile.rollup"""
+        }
+    }
+    docker.withRegistry("${env.REGISTRY}, '${env.REGISTRY_CREDENTIALS}') {
```

Comment:
Looks like this is missing an end quote.

Reply:
Also `s/'/"/` here and for `IMAGE_NAME` below -- IIRC groovy won't sub in vars in single quotes, right?

Reply:
Fixed all that, thanks!  I also set up the env vars in my test job and did a run :arrow_down: